src/vopr.zig

```diff
@@ -1041,23 +1041,17 @@ pub const Simulator = struct {
             // If the entire Zone.wal_headers is corrupted, the replica becomes permanently
             // unavailable (returns `WALInvalid` from `open`). In the simulator, there are only two
             // WAL sectors, which could both get corrupted when a replica crashes while writing them
-            // simultaneously. To keep the replica operational, arbitrarily repair one of the
-            // sectors.
+            // simultaneously. Repair both sectors so that even if one of them corrupted on startup,
+            // the replica still remains operational.
```

Comment:
```suggestion
            // simultaneously. Repair both sectors so that even if one of them becomes corrupted on startup,
            // the replica still remains operational.
```